### PR TITLE
Adding arccosh_  to torch frontend

### LINKED ISSUE #14138: arccosh_

### PR DESCRIPTION
**Pull request by vtsamit**
Close #14138

### CODE REVIEW

**Comment by Infrared1029** [comment]
Hi @vtsamit, thanks a lot for your contribution, the tests are failing because you are calling `self.arccosh`, we do not have the instance method for `arccosh` in our torch `Tensor` class, use `self.acosh` instead, then make sure the tests are passing:)!

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -400,6 +400,11 @@ def acos_(self):
         self._ivy_array = self.acos().ivy_array
         return self
 
+    @with_unsupported_dtypes({"1.11.0 and below": ("float16",)}, "torch")
+    def arccosh_(self):
+        self._ivy_array = self.arccosh().ivy_array
```
Comment (Infrared1029) [inline ivy/functional/frontends/torch/tensor.py:405]:
```suggestion
        self.ivy_array = self.arccosh().ivy_array
```

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by vtsamit** [comment]
Thank you sir for letting me know. I have made the required changes.

**Comment by Infrared1029** [comment]
lgtm!

**Pull request merged**